artifact/README.md

```diff
@@ -0,0 +1,39 @@
+# Building & Using Artifact Container
+
+The `Dockerfile` in this directory creates a ready-to-use Docker container with compiled versions of Gobra.
+The full steps are as follows:
+1. Build the image:
+    ```
+    docker build -t gobraverifier/gobra-artifact:v1 .
+    ```
+   This downloads the specified versions of Gobra, Silver, Silicon, and Carbon into the image and compiles them to `/home/gobra/gobra.jar` and `/home/gobra/gobra-test.jar`.
+   `gobra.jar` just contains an executable version of Gobra whereas `gobra-test.jar` also includes the test classes. 
+2. Start the container:
+    - The following command starts an interactive shell in the container:
+        ```
+        docker run -it gobraverifier/gobra-artifact:v1
+        ```
+    - If you want to share a folder between your host and the container, use the following command that creates a new folder in the current directory on your host and mounts it to the container (it will be mounted at `/home/gobra/sync`):
+        ```
+        docker run -it --volume $PWD/sync_$(date +%Y-%m-%d_%H-%M):/home/gobra/sync gobraverifier/gobra-artifact:v1
```

Comment:
The command does not work on windows (because of the date). I think that you should describe what the `$PWD/sync_$(date +%Y-%m-%d_%H-%M)` is for. You should also describe where the folder is created in the host machine.

Reply:
Good to know. However, this README is not so important since it is more for documentation purposes for us and is not copied into the docker image (as opposed to `artifact/content/README.md`)